test/controllers/update/update.beneficial.owner.type.controller.spec.ts

```diff
@@ -0,0 +1,134 @@
+jest.mock("ioredis");
+jest.mock('../../../src/middleware/authentication.middleware');
+jest.mock('../../../src/middleware/service.availability.middleware');
+jest.mock('../../../src/utils/application.data');
+jest.mock('../../../src/middleware/navigation/update/has.entity.update.middleware');
+
+import { describe, expect, test, beforeEach, jest } from '@jest/globals';
+import { NextFunction, Request, Response } from "express";
+import request from "supertest";
+
+import app from "../../../src/app";
+import { authentication } from "../../../src/middleware/authentication.middleware";
+import { serviceAvailabilityMiddleware } from "../../../src/middleware/service.availability.middleware";
+import { hasEntityUpdateDetails } from "../../../src/middleware/navigation/update/has.entity.update.middleware";
+import * as config from "../../../src/config";
+import { getApplicationData } from '../../../src/utils/application.data';
+import {
+  SERVICE_UNAVAILABLE,
+  BENEFICIAL_OWNER_TYPE_PAGE_HEADING_ALL_IDENTIFIED_ALL_DETAILS,
+  BENEFICIAL_OWNER_TYPE_LEGEND_TEXT_ALL_IDENTIFIED_ALL_DETAILS,
+  BENEFICIAL_OWNER_TYPE_ADD_BUTTON_ALL_IDENTIFIED,
+  BENEFICIAL_OWNER_TYPE_PAGE_GOVERNMENT_BO,
+  BENEFICIAL_OWNER_TYPE_PAGE_CORPORATE_BO,
+  BENEFICIAL_OWNER_TYPE_PAGE_CORPORATE_MO,
+  BENEFICIAL_OWNER_TYPE_PAGE_INDIVIDUAL_BO,
+  BENEFICIAL_OWNER_TYPE_PAGE_INDIVIDUAL_MO,
+  PAGE_TITLE_ERROR
+} from '../../__mocks__/text.mock';
+import {
+  APPLICATION_DATA_MOCK,
+  ERROR
+} from '../../__mocks__/session.mock';
+import { ErrorMessages } from '../../../src/validation/error.messages';
+import { BeneficialOwnersStatementType, BeneficialOwnerStatementKey } from '../../../src/model/beneficial.owner.statement.model';
+import { BeneficialOwnerTypeChoice, BeneficialOwnerTypeKey } from '../../../src/model/beneficial.owner.type.model';
+
+const mockAuthenticationMiddleware = authentication as jest.Mock;
+mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
+mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockHasEntityUpdate = hasEntityUpdateDetails as jest.Mock;
+mockHasEntityUpdate.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockGetApplicationData = getApplicationData as jest.Mock;
+
+describe("BENEFICIAL OWNER TYPE controller", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe("GET tests", () => {
+    test("renders the beneficial owner type page for beneficial owners", async () => {
```

Comment:
would it make more sense to call the name of the page from the configs instead of hard coding it here?